app/views/deploys/changeset.html.erb

```diff
@@ -10,7 +10,7 @@
   <%= render 'changeset/pulls', changeset: @changeset %>
 </div>
 
-<div class="tab-pane" id="risks">
+<div class="tab-pane" id="risks" data-risky="<%= @changeset.risks? ? 1 : 0 %>">
```

Comment:
How about adding this to the tab link itself? And maybe making it generic (like an `active` class or `data-active`)?